src/cloud-api-adaptor/podvm/Dockerfile.podvm_builder.rhel

```diff
@@ -9,6 +9,9 @@
 FROM registry.access.redhat.com/ubi9/ubi:9.2
 
 ARG ARCH
+# PROTOC_ARCH="x86_64" | "s390x_64"
+ARG PROTOC_ARCH="x86_64"
+ARG YQ_ARCH
```

Comment:
Is YQ_ARCH being set?

Reply:
in the makefile

Reply:
Should we add it to the docs as well: https://github.com/confidential-containers/cloud-api-adaptor/blob/main/src/cloud-api-adaptor/podvm/README.md?
(or reconcile the fact that the docs suggest a different process to the `make podvm-builder` that you are expecting 🙃)

Reply:
right, AFAIU the `make podvm-builder` is just the quick way, i think we surly need to describe how to use local file in the advance way as well. I'll update it.